examples/with-redis/package.json

```
@@ -1,18 +1,17 @@
 {
-  "name": "with-redis",
+  "name": "roadmap-voting",
```

```suggestion
  "name": "with-redis",
```

Let's keep this the same as the folder name